supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -325,7 +324,15 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     }
   }
 
-  override def pollStatusAsync(handle: JesPendingExecutionHandle): Future[RunStatus] = super[JesStatusRequestClient].pollStatus(handle.runInfo.get)
+  override def pollStatusAsync(handle: JesPendingExecutionHandle): Future[RunStatus] = {
+    super[JesStatusRequestClient].pollStatus(handle.runInfo.get) map {
+      case RunStatus.Success(eventList, machineType, zone, instanceName) if jobDescriptor.key.attempt == 1 =>
```

Comment:
oh oops, this 100% should not be left in...

that's the stuff to see what happens if every first task gets preempted 😳